build.sbt

```diff
@@ -13,19 +13,18 @@ scalacOptions ++= Seq(
   "-Ywarn-unused-import")
 
 val awsVersion = "1.11.234"
-val playVersion = "2.5.18"
+val playVersion = "1.1.2"
```

Comment:
this is a big jump backwards, is this correct?

Reply:
The ws and json libraries that play supplies were split out into independent libraries so the numbers have dropped https://github.com/playframework/play-ws